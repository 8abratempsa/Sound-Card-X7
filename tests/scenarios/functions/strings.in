octosql "SELECT 'test' LIKE '%es%' as line_1, 'test' LIKE 'es', 'test' LIKE 'tes%', 'test' LIKE '%est', 'test' LIKE 'es%', 'test' LIKE 't_st', 'test' LIKE 't_t',
                'test' ~ '^te.*' as line_2, 'test' ~ '^tE.*', 'test' ~ 'te.*', 'test' ~ 'es', 'test' ~ '^es$',
                'test' !~ '^te.*' as line_3, 'test' !~ '^tE.*', 'test' !~ 'te.*', 'test' !~ 'es', 'test' !~ '^es$',
                'test' ~* '^te.*' as line_4, 'test' ~* '^tE.*', 'test' ~* 'te.*', 'test' ~* 'es', 'test' ~* '^es$',
                'test' !~* '^te.*' as line_5, 'test' !~* '^tE.*', 'test' !~* 'te.*', 'test' !~* 'es', 'test' !~* '^es$',
                upper('test') as line_6, upper('TEST'),
                lower('test') as line_7, lower('TEST'),
                reverse('test') as line_8,
                substr('test', 1) as line_9, substr('test', 1, 2),
                replace('test', 'es', 'ESS') as line_10,
                position('test', 'es') as line_11, position('test', ''), position('test', 'ex'), position('test', 'testt'), position('test', 'test'),
                len('test') as line_12"
