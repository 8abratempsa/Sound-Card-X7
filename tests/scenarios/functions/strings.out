+--------+-------+-------+-------+-------+-------+-------+--------+-------+-------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+---------+---------+--------+--------+--------+--------+---------+
| line_1 | col_1 | col_2 | col_3 | col_4 | col_5 | col_6 | line_2 | col_8 | col_9 | col_10 | col_11 | line_3 | col_13 | col_14 | col_15 | col_16 | line_4 | col_18 | col_19 | col_20 | col_21 | line_5 | col_23 | col_24 | col_25 | col_26 | line_6 | col_28 | line_7 | col_30 | line_8 | line_9 | col_33 | line_10 | line_11 | col_36 | col_37 | col_38 | col_39 | line_12 |
+--------+-------+-------+-------+-------+-------+-------+--------+-------+-------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+---------+---------+--------+--------+--------+--------+---------+
| true   | false | true  | true  | false | true  | false | true   | false | true  | true   | false  | false  | true   | false  | false  | true   | true   | true   | true   | true   | false  | false  | false  | false  | false  | true   | 'TEST' | 'TEST' | 'test' | 'test' | 'tset' | 'est'  | 'es'   | 'tESSt' |       1 |      0 | <null> | <null> |      0 |       4 |
+--------+-------+-------+-------+-------+-------+-------+--------+-------+-------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+---------+---------+--------+--------+--------+--------+---------+
